apps/dashboard/src/components/workflow-editor/steps/configure-step-template-issue-cta.tsx

```diff
@@ -26,19 +24,63 @@ export const ConfigureStepTemplateIssueCta = (props: ConfigureStepTemplateIssueC
           <span>Help?</span>
         </ExternalLink>
       </div>
-      <Link to={'./edit'} relative="path" state={{ stepType: step.type }}>
-        <Button
-          size="sm"
-          variant="secondary"
-          mode="outline"
-          className="flex w-full justify-start gap-1.5 text-xs font-medium"
-          type="button"
-        >
-          <span className="bg-destructive h-4 min-w-1 rounded-full" />
-          <TruncatedText>{issue}</TruncatedText>
-          <RiArrowRightUpLine className="text-destructive ml-auto h-4 w-4" />
-        </Button>
-      </Link>
+      {children}
     </SidebarContent>
   );
 };
+
+type ConfigureStepTemplateContentIssueCtaProps = {
+  step: StepResponseDto;
+  issue: StepContentIssue;
+};
+export const ConfigureStepTemplateContentIssueCta = (props: ConfigureStepTemplateContentIssueCtaProps) => {
+  const { step, issue } = props;
+
+  return (
+    <Link to={'./edit'} relative="path" state={{ stepType: step.type }}>
+      <Button
+        size="sm"
+        variant="secondary"
+        mode="outline"
+        className="flex h-full w-full justify-start gap-3 py-2 text-xs"
+        type="button"
+      >
+        <span className="bg-destructive h-full min-w-1 rounded-full" />
+        <div className="flex flex-col items-start gap-0.5">
+          <TruncatedText className="font-medium">Invalid variable: {issue.variableName}</TruncatedText>
+          <p className="text-text-soft text-wrap text-start">{issue.message}</p>
+        </div>
+        <RiArrowRightUpLine className="text-destructive mb-auto ml-auto h-4 w-4" />
+      </Button>
+    </Link>
+  );
+};
+
+type ConfigureStepTemplateIntegrationIssueCtaProps = {
+  step: StepResponseDto;
+  issue: StepIntegrationIssue;
+};
+export const ConfigureStepTemplateIntegrationIssueCta = (props: ConfigureStepTemplateIntegrationIssueCtaProps) => {
+  const { step, issue } = props;
+
+  return (
+    <Link to={'/integrations'} relative="path" state={{ stepType: step.type }}>
```

Comment:
Consider making this dryer. The UI is almost identical. Also, as far as I can tell from the screenshot the black arrow is smaller from the red arrow.